doc/whats_new.rst

```diff
@@ -205,6 +205,12 @@ Bug
 
 - Fix :meth:`mne.io.Raw.interpolate_bads` for interpolating CTF MEG channels when reference sensors are present by `jeythekey`_
 
+- Fix :meth:`mne.preprocessing.ICA.score_sources` to use the ``sfreq`` of the raw data to filter rather than the ``sfreq`` when the ICA was fit in :func:`mne.preprocessing.ICA._band_pass_filter` by `jeythekey`_
```

Comment:
we don't document in what's new the private functions. These are not visible to end users. Only document changes to public API. thx